examples/Bunch/cmakeFlags

```diff
@@ -35,6 +35,16 @@ flags[2]="-DCUDA_ARCH=35 -DPARAM_OVERWRITES:LIST=-DPARAM_INCLUDE_FIELDBACKGROUND
 flags[3]="-DCUDA_ARCH=35 -DPARAM_OVERWRITES:LIST=-DPARAM_SINGLE_PARTICLE=true"
 flags[4]="-DCUDA_ARCH=35 -DPARAM_OVERWRITES:LIST=-DENABLE_SYNCHROTRON_PHOTONS=1"
 flags[5]="-DCUDA_ARCH=35 -DPARAM_OVERWRITES:LIST=-DPARAM_FILTER_GAMMA=1"
+flags[6]="-DCUDA_ARCH=35 -DPARAM_OVERWRITES:LIST=-DPARAM_RADWINDOW=radWindowFunctionTriangle"
```

Comment:
you can also combine the tests and test your form factors with the window functions